### PR TITLE
Interpolate and cleanup

### PR DESCRIPTION
**Pull request by Thomas-Geraghty**
- Switch all currently used views to use (excluding steps, in it's own PR as part of add-steps) to use the new interpolate function 
- Moved all hardcoded text strings into locale files, put variables in place
- Bit of general cleanup, removed some unnecessary 'isPageHeading' attributes, unused controller/views, small fix for agree-plan page.

Note-  as usual, all broken up by commit, might be easier to review that way.

### CODE REVIEW

File: server/routes/aboutPop/locale.json [hunk 1]
```diff
@@ -1,6 +1,11 @@
 {
   "en": {
-    "title": "About {POP_NAME}",
+    "page": {
+      "title": "About {{ subject.givenName }} - {{ applicationName }}"
```
Comment (pipwilson) [inline server/routes/aboutPop/locale.json]: This should just say `About - {{ applicationName }}`

Comment (Thomas-Geraghty) [reply server/routes/aboutPop/locale.json]: Nice one, couldn't see anything on the spreadsheet for it, but this makes sense as we're doing no PII in the titles

Comment (Thomas-Geraghty) [reply server/routes/aboutPop/locale.json]: Resolved https://github.com/ministryofjustice/hmpps-sentence-plan-ui/pull/113/commits/b4b6e2ceaf7c4fe8234609879588600eb174c1b4

**Comment by pipwilson** [comment]
I think this all looks good but I think the "create step" page is missing.

**Review by pipwilson — APPROVED** [review]
LGTM. Big improvement.

**Pull request merged**